charts/sonarqube-dce/values.yaml

```diff
@@ -344,6 +344,32 @@ ApplicationNodes:
   ##
   extraContainers: []
 
+  # Define the Horizontal Pod Autoscaler for the app deployment
+  hpa:
+    enabled: false
+    minReplicas: 2
+    maxReplicas: 10
+    metrics:
+      - type: Resource
+        resource:
+          name: cpu
+          target:
+            type: Utilization
+            averageUtilization: 80
+    behavior:
+      scaleDown:
+        stabilizationWindowSeconds: 60
+        policies:
+          - type: Pods
+            value: 1
+            periodSeconds: 0
+      scaleUp:
+        stabilizationWindowSeconds: 60
+        policies:
+          - type: Pods
+            value: 1
+            periodSeconds: 0
```

Comment:
also when trying to install it by only changing `enabled:true` I get this error
`Error: UPGRADE FAILED: failed to create resource: HorizontalPodAutoscaler.autoscaling "toto-sonarqube-dce-app-hpa" is invalid: [spec.behavior.scaleUp.policies[0].periodSeconds: Invalid value: 0: must be greater than zero, spec.behavior.scaleDown.policies[0].periodSeconds: Invalid value: 0: must be greater than zero] `